### ISSUE TITLE
Reduce travel required for 3-finger gestures

### ISSUE THREAD

**Issue by cacarr-pdxweb** [mpiannucci/gnome-shell-extended-gestures#19]
I'm only able to trigger the toggle overview action on every 3rd or 4th attempt at a 3 finger vertical gesture. I'm on a 4th gen X1 Carbon.

I would like to be able to reduce the amount of travel required to trigger the action, as it seems to me that I should be able to just quickly close my hand with 3-fingered vertical flick, rather than nearly having to move my entire arm.

I would use the gesture all the time, if it didn't require so much effort.

**Comment by mpiannucci** [comment]
Okay I can look into adding some more settings for sensitivity. 

Thanks for reporting

**Comment by kevinlaugh** [comment]
First, thank you for this great and much needed extension. It will definitely be one of the "must have" extensions for every laptop with a precision touchpad. 
I just wanted to report that on this issue, my touchpad does not experience this issue. All gestures are very minute finger travel. So this issue may be hardware or system setting specific.
Laptop: Dell XPS 9550 (i7 6th Gen, 32GB)
OS: Ubuntu 17.10, clean install using default Gnome config
Gnome Version: 3.26

**Comment by mpiannucci** [comment]
Thanks @kevinlaugh 

Are you saying that it is too sensitive on your laptop? Sensitivity is the next thing on my radar

**Comment by tttom64** [comment]
Even with added settings, the travel should be drastically lowered to match that of GNOME's default 4-finger gestures. Those are effortless and easy to use, and trigger almost every time I try them. The three-finger gestures from the extension on the other hand, are frustrating and only work if I carefully drag my fingers slowly across the entire touchpad.

**Comment by mpiannucci** [comment]
Yeah it definitely needs to be a settings because it matches the default gesture really well on my macbook pro. So its very hardware dependent. This is my next feature to work on.

**Comment by mpiannucci** [comment]
So i think the motion threshold is thrown off becasue we have to do more direction checking than gnome does by default becasue they only care about vertical gestures. Our extra checking decreses the sensitivity on some laptops for sure. 

That is probably something else to look into, but for now I have added a new setting to change the sensitivity using the settings GUI. You can set the threshold for vertical and horizontal gestures to be more sensitive by increasing the values and less sensitive by decreasing the values. 

let me know how this works out for anyone who can test it.

**Comment by mpiannucci** [comment]
I doubled the allowed sensitivity adjustment for my next feature merge so hopefully that will help some people
